src/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -157,10 +159,10 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
     std::shared_ptr<ngraph::Function> functionRefs;
     std::map<std::string, std::string> configuration;
     // Non default values of layouts/precisions will be set to CNNNetwork
-    InferenceEngine::Layout inLayout = InferenceEngine::Layout::ANY;
-    InferenceEngine::Layout outLayout = InferenceEngine::Layout::ANY;
-    InferenceEngine::Precision inPrc = InferenceEngine::Precision::UNSPECIFIED;
-    InferenceEngine::Precision outPrc = InferenceEngine::Precision::UNSPECIFIED;
+    std::vector<InferenceEngine::Layout> inLayout;
```

Comment:
`std::vector<InferenceEngine::Layout> inLayout = { InferenceEngine::Layout::ANY };`
The same for other ones to avoid to create a unnecessary long vector in the constructor

Reply:
Fixed in commit https://github.com/openvinotoolkit/openvino/pull/9975/commits/f7b420d352fd5e02ccf3f472b9f93a88f4f8d252 and https://github.com/openvinotoolkit/openvino/pull/9975/commits/033cbb9f670ea0f4d5050995eeaf98e84280f31b.